syntax = "proto2";

package e2e_noa;

enum ALCState {
  ALC_OFF = 1;
  ALC_STANDBY = 2;
  ALC_STANDBY_ENABLE = 3;
  ALC_ONGOING = 5;
  ALC_COMPLETED = 6;
  ALC_CROSSING_LANE = 7;
  ALC_RETURNING = 8;
  ALC_RETURN_COMPLETED = 9;
  ALC_PREPARE = 10;

  reserved 4;
}

enum LaneChangeDirection {
  LCD_NONE = 1;
  LCD_LEFT = 2;
  LCD_RIGHT = 3;
}
